lib/db_query_matchers/make_database_queries.rb

```diff
@@ -46,6 +49,9 @@ def pluralize(count, singular, plural = nil)
     if options[:manipulative]
       counter_options[:matches] = [/^\ *(INSERT|UPDATE|DELETE\ FROM)/]
     end
+    if options[:unscoped]
+      counter_options[:matches] = [/^ SELECT(?!\sCOUNT).*FROM(?!.*(WHERE|LIMIT))/mx]
```

Comment:
With this regex focused on `SELECT` statements, we're making the `unscoped` option mutually exclusive to the `manipulative` option. 

Is that intentional? Could we potentially write this to allow any of `INSERT` / `UPDATE` / `DELETE` statements that are unscoped?